conftest.py

```diff
@@ -13,7 +36,30 @@ def pytest_runtest_makereport(item, call):
                 run_command()
 
 def run_command():
+    '''
```

Comment:
This is interesting.  Can you let me know if you found pytests logging feature missing something  you're balancing with this function?  I think there was a way to throw the console output out when a test fails, though I used that feature a while ago.... anyhow, ya, let me know the genesis for this, I'd be interested to know what you were not seeing.

Reply:
@EfrainOlivaresUVA , I believe [this commit](https://github.com/tslever/wordprocessors/commit/8a1a1d72543ccb06baf2e3cac8a6879609aeb740) shows that I tried to print a pytest log if tests fail using `validations.yml` but used `conftest.py` instead. I believe I didn't see a message that tests had failed in the output of the GitHub action associated with `validations.yml`.